wiki/History_of_osu!/2010/en.md

```diff
@@ -1 +1,79 @@
+---
+stub: true
+---
+
 # History of osu! 2010
+
+## January
+
+The [Wai-con](https://en.wikipedia.org/wiki/Wai-Con) anime convention in Perth had osu!arcade, a fully self-contained arcade machine running a touch-based version of osu!. Although playing too much on it could lead to sore fingers as some people had criticised, the prototype containing custom-made hardware and a built-in touchscreen was well-received by the community.[^wai-con]
+
+![A touchscreen displaying osu!'s main menu](img/osu!arcade.jpg "The osu!arcade machine at 2010's Wai-con")
+
+## March
+
+Through February to March, [peppy](https://osu.ppy.sh/users/2) organised an art competition dedicated to banners for in-game challenges, with the prize being the art's official recognition and a year of [osu!supporter](/wiki/osu!supporter) or physical goods. After a [community voting](https://osu.ppy.sh/community/forums/topics/27112?n=1) round, the osu! team made the final choice.[^art-challenge]
+
+On March 8, an osu! update brought considerable performance and audio improvements to the game.[^stable-b1485]
+
+## April
+
+Due to infrastructure changes, the [IRC](https://en.wikipedia.org/wiki/Internet_Relay_Chat) server was moved to the address `irc.ppy.sh`.[^irc]
+
+This month, peppy introduced a new star rating difficulty algorithm in an osu! update.[^stable-b1596]
+
+## August
+
+On August 2, all users were granted a single username change. To change username, they had to have received an osu!supporter tag at least once, either through purchase or as a gift.[^name-change]
+
+The [`!report`](/wiki/BanchoBot#report) chat command was added to [Bancho](/wiki/BanchoBot), which allowed people to notify chat moderators about users misbehaving in the chat.[^bancho-report]
+
+[nardii](https://osu.ppy.sh/users/1017) gave peppy access to their [Twitter](https://twitter.com) account [@osugame](https://twitter.com/osugame) which was used for general osu! news from then on.[^twitter-osugame]
+
+## September
+
+On September 2, osu! received several mapping updates, including:[^stable-b1650]
+
+- The [SB load](/wiki/Client/Beatmap_editor/SB_load) indicator, reflecting drawing efficiency of storyboard and background elements.
+- The introduction of [Approach Rate](/wiki/Beatmap/Approach_rate) for configuring the speed of [approach circles](/wiki/Gameplay/Hit_object/Approach_circle) independently of [Overall Difficulty](/wiki/Beatmap/Overall_difficulty).
+- Custom [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) multipliers ranging from 0.5x to 2x.
+
+60 [difficulties](/wiki/Beatmap/Difficulty) were affected by a Taiko bug causing drum-rolls to be incorrectly displayed and thus, all scores on them were deleted.[^taiko-reset]
```

Comment:
The reference of `Taiko` here does count as a historic mention, doesn't it? This writing style is also used in the forum threads.

Out of curiousity, when was the writing style changed to `osu!taiko` and `osu!catch`, and should this be reflected in the wiki, like described in the [ASC § osu!](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#osu!)?

Reply:
as in the game mode name was "Taiko" back then? if that's the case, it can probably stay as is

may want to explain somehow that it's the old name though